data-lineage/src/main/java/com/wipro/ats/bdre/lineage/dataaccess/LineageDao.java

```diff
@@ -126,9 +126,9 @@ public static void closeConnection() throws SQLException {
 //			throw new ETLException("Table " + tableName + " does not exist in Hive db");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)